### DISCUSSION TITLE
win11 修改 hosts 文件需要管理员权限 | /project/win11-hosts/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#589] [Announcements]
https://yuanfan.rbind.io/project/win11-hosts/

**Comment by CyrusYip** [comment]
是哪个 AI 教那么复杂的办法？我说个简单的：按 Win 键，输入记事本，右击，以管理员身份运行，打开文件，编辑保存。

**Reply by earfanfan** [reply]
哈哈，是字节跳动家的豆包<https://www.doubao.com/chat>，教了我好几种方法都不行，最后才试出来那么干的。

你这个简单方法我试了试，是可行的。
